test/com/sun/jna/StructureTest.java

```diff
@@ -227,6 +228,11 @@ protected void ensureAllocated() {
             return FIELDS;
         }
     }
+    @FieldOrder({ "field0", "field1" })
+    public static class EasyTestStructure0 extends FilledStructure {
```

Comment:
The name is not optimal - "Easy" is a personal evaluation - I think "Annotated" might fit better.

Reply:
**fixed**